README.md

```diff
@@ -275,14 +274,9 @@ by adding the `aws-crt` gem to your Gemfile.
 is an extension to Sigv4 that allows signatures that are valid in more than one region.
 Sigv4a is required to use some services/operations such as
 [S3 Multi-Region Access Points](https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiRegionAccessPoints.html)
-and Amazon EventBridge Global Endpoints.
-Currently sigv4a requires the [aws-crt](https://rubygems.org/gems/aws-crt/) gem.  The `aws-sigv4` signers 
-will automatically use the CRT provided signers with support for `sigv4a` when the `aws-crt` gem is available.
-
-```ruby
-gem 'aws-sdk-s3', '~> 1'
-gem 'aws-sigv4', '1.4.1.crt'
-```
+and Amazon EventBridge Global Endpoints.  Sigv4a is supported natively in Ruby
```

Comment:
May want to rewrite this entire section about how CRT only speeds up sigv4a and mentioning what sigv4a is used for seems out of place and not necessary. It should be transparent to customers.

Reply:
CRT sigv4a is currently slower with ffi bindings, so didn't want to mention it.  I can move the sigv4a section into its own section elsewhere in the README or just get rid of it.  Thoughts?

Reply:
Whatever you'd like, my comment was to raise awareness that this section may or may not even be necessary anymore.

Reply:
I think the sigv4a section in CRT can probably just be removed. We already say how to install it and why (crc32c)